packages/kbn-i18n/src/core/i18n/index.js

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
__question:__ btw, why we do we need `core/i18n` subfolder, can't we use just `core`?

Reply:
I did it because of next reason: this functionality is used in `Angular` and `React` wrappers, and here we should also split into `browser` and `node` implementation as `React` for instance is the same for both. Also, I split real core from loader, where we do not have any environment specific code.